### PR TITLE
Sd 3790 get workflow by

### PR DESCRIPTION
**Pull request by alpeshjikadra**

### CODE REVIEW

File: src/test/java/com/kylas/sales/workflow/stubs/WorkflowStub.java [hunk 1]
```diff
@@ -24,13 +26,47 @@ public static WorkflowRequest anEditPropertyWorkflowRequest(
...
+    allowedActions.setWrite(canCreate);
+    WorkflowDetail workflowDetail = new WorkflowDetail();//id, name, description, entityType,workflowTrigger,condition,actions, allowedActions);
+    workflowDetail.setId(id);
+    workflowDetail.setName(name);
+    workflowDetail.setDescription(description);
+    workflowDetail.setEntityType(entityType);
+    workflowDetail.setTrigger(workflowTrigger);
+    workflowDetail.setCondition(condition);
+    workflowDetail.setActions(actions);
+    workflowDetail.setAllowedActions(allowedActions);
```
Comment (abhishek-bhuskute) [inline src/test/java/com/kylas/sales/workflow/stubs/WorkflowStub.java:67]: Should we have a constructor or builder for this?

**Review by abhishek-bhuskute — APPROVED** [review]

File: src/test/java/com/kylas/sales/workflow/stubs/WorkflowStub.java (hunk 1 above)
Comment (alpeshjikadra) [reply src/test/java/com/kylas/sales/workflow/stubs/WorkflowStub.java:67]: Earlier I add constructor, but it's just a value object so I removed constructor and add setter, But Yes I can try something new let me check

File: src/main/java/com/kylas/sales/workflow/api/response/WorkflowDetail.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+//    this.entityType = entityType;
+//    this.trigger = trigger;
+//    this.condition = condition;
+//    this.actions = actions;
+//    this.allowedActions = allowedActions;
+//    this.createdBy = createdBy;
+//    this.updatedBy = updatedBy;
+//    this.createdAt = createdAt;
+//    this.updatedAt = updatedAt;
+//  }
```
Comment (abhishek-bhuskute) [inline src/main/java/com/kylas/sales/workflow/api/response/WorkflowDetail.java:48]: The constructor was there earlier.  We can use Lombok builder if there are too many optional properties.

File: src/test/java/com/kylas/sales/workflow/stubs/WorkflowStub.java (hunk 1 above)
Comment (abhishek-bhuskute) [reply src/test/java/com/kylas/sales/workflow/stubs/WorkflowStub.java:67]: This is something you will like- https://projectlombok.org/features/Builder

**Pull request merged**